Tests/PackageLoadingTests/PackageBuilderTests.swift

```diff
@@ -41,9 +41,12 @@ class PackageBuilderTests: XCTestCase {
     }
 
     func testMixedSources() throws {
+        let foo: AbsolutePath = AbsolutePath("/Sources/foo")
```

Comment:
Quick question, was the repetition of the type needed, or just stylistic?  Just curious — in theory you just need the `=` for all of these, not the `:`'.

Reply:
Just stylistic; I tend to use explicit types for my own code, and it has become habitual.  I don't mind dropping the type annotations if you prefer.

Reply:
In SwiftPM we've tended to not spell out types (to the point of using `.init()` instead of mentioning the type when it's obvious, which took me some time to get used to).  If you don't mind doing that it would be great, but I don't think it's a huge deal.  I think @tomerd runs swift-format on his changes, I don't know if it would catch this.

In any case, I realize this is a fairly trivial comment compared to the improvements of this PR.

Reply:
yea `let foo = AbsolutePath("/Sources/foo")` is the most idiomatic afaik